backend/diagnostics.go

```diff
@@ -31,5 +31,5 @@ type CheckHealthRequest struct {
 type CheckHealthResult struct {
 	Status      HealthStatus
 	Message     string
-	JSONDetails string
+	JSONDetails []byte
```

Comment:
can this be `interface{}` and the SDK will convert it to bytes for me? 

In the frontend code... we assume this is an optional object:
https://github.com/grafana/grafana/blob/master/packages/grafana-runtime/src/utils/DataSourceWithBackend.ts#L25

Reply:
`interface{}` is not a standard protobuf type.

But we can definitely convert it to `interface{}` in the Grafana before we send it to the frontend.